tck/pom.xml

```diff
@@ -25,7 +25,7 @@
     <parent>
         <groupId>org.eclipse.microprofile.reactive.messaging</groupId>
         <artifactId>microprofile-reactive-messaging-parent</artifactId>
-        <version>2.1-SNAPSHOT</version>
+        <version>3.0-SNAPSHOT</version>
```

Comment:
The arquillian dependency management is incorrect. The correct version has been specified in the parent pom.xml. You just need to declare the dependency on the api and remove the line 35 and 39-40